docs/sqllab.rst

```diff
@@ -122,3 +122,19 @@ database configuration:
 
 Here, "version" should be the version of your Presto cluster. Support for this
 functionality was introduced in Presto 0.319.
+
+Create Table As
+'''''''''''''''
+
+You can use `CREATE TABLE AS SELECT ...` statements on SQLLab this feature can be toggled on
+and off at the database configuration level, note that on PostgreSQL DDL is transactional,
+this means that, to properly use this feature you have to set `autocommit` to true on your
+engine parameters:
```

Comment:
Perhaps clarify by explicitly mentioning that CTAS require DML + split up into two sentences to make it easier on the eyes.

Reply:
My bad; I somehow assumed this was being done as DML, not using the button! The text above sounds gook. Not sure if this is overkill, but it might be a good idea to also mention this under `docs/installation.rst` where database-specific quirks are listed, or at least a reference from one to the other.

Reply:
no problem, I would say that a small note is not overkill. Just updated it

Reply:
^^ @villebro